scripts/fedbiomed_run

```diff
@@ -242,11 +242,11 @@ case $1 in
                 CONTAINER_UID=$(id -u) CONTAINER_GID=$(id -g) \
                              CONTAINER_USER=$(id -un | sed 's/[^[:alnum:]]/_/g') \
                              CONTAINER_GROUP=$(id -gn | sed 's/[^[:alnum:]]/_/g') \
-                             docker-compose build restful mqtt
+                             docker compose build restful mqtt
```

Comment:
Hi @mvesin,

After merging this branch into develop/master the installations that have old version of docker won't work anymore. Do you think that it is good idea to add deprecation warning instead of failing. For example hte script can be modified to support following scenarios:

- user has old docker version  -> raise deprecation warning and ask for installing version and run `docker-compose`
- user has newer version -> use `docker compose`

I know that it is also the same for conda packages.  It would be nice to have versioning for conda yml files that we can check which version of conda env is in use. Because the users who will pull from latest and have already a older conda envs will face problems.  We do not have mechanism that checks the version of the conda env. Of course it may be overkill for this merge request. However, if we assume that the Fed-BioMed will distributed in package managers such as conda or pip it won't be problem since each time Fed-BioMed upgraded or installed the packages compatible with the version will be reinstalled or upgraded as well. 

However, to me, docker is more independent than conda environments considering that the user may also use docker for other applications. Conda is more for having isolated environment which is fine to ask user upgrade your conda environment.

Reply:
Thank you @srcansiz for the comment & review:
- regarding versioning of conda envs: I did not implement yet (might not be that simple, raises questions about what we exactly want to do/check), but created issue #808 for it
- regarding `docker compose` migration smoothing for users, I added in (hopefully) all scripts that use the command
  * backward compatibility: if no `docker compose` is found, it raises a warning and uses `docker-compose`. I'm not sure it is a good idea as we mentionned that we don't support it but that may ease the short term life of users. Mid-term, we should probably remove this backward compatibility additions and keep only `docker compose` (and fail if not existing)
  * failure with error message if no `docker compose` or `docker-compose` is found. Exception: the `fedbiomed_environment clean` will just skip docker container cleaning, and continue with as much cleaning as possible to fit the spirit of this script